redditdownload/gfycat.py

```diff
@@ -23,17 +23,17 @@ def __init__(self):
         super(gfycat, self).__init__()
 
     def __fetch(self, url, param):
-        import urllib2
+        import urllib.request, urllib.error, urllib.parse
```

Comment:
Any in-function imports were there before I started making changes, but I'll fix if I see them.